tests/publiccloud/xfsprepare.pm

```diff
@@ -0,0 +1,79 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Prepare secondary disk for usage in xfstests
```

Comment:
I bet the test is doing more than preparing a disk :)

Reply:
Yes, it also installs the `xfstests` but not much more :slightly_smiling_face: 

(I updated the Summary)

Reply:
The actual test is from `tests/xfstests/run` and we could re-use it, which is nice :slightly_smiling_face:

Reply:
ah ok, I was actually wondering where is the "actual test" run (facepalm) :)